e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -144,10 +144,9 @@ $certificateHashAlgorithm = "SHA256"
 
 if ($GenerateResourcesForDevOpsPipeline)
 {
-    $iothubUnitsToBeCreated = 3;
+    $iothubUnitsToBeCreated = 5;
```

Comment:
Since we've been having throttling issues, we've manually increased this to 5, so changing script to match.